### PR TITLE
update license information

### PR DESCRIPTION
**Pull request by nlu90**
Attempt to add and update license inforamtion.

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Apache Incubator-Heron
+Apache Heron
```
Comment (nwangtw) [inline NOTICE]: is this expected?

Comment (nlu90) [reply NOTICE]: Sijie suggested we should make this change in the slack channel

**Comment by nlu90** [comment]
@sijie  Could you help take a look whenever you have some time?

File: LICENSE [hunk 2]
```diff
@@ -199,3 +199,43 @@
...
+    (MIT License) Bootstrap(v3.1.1) (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+    (MIT License) JQuery (https://jquery.org/license/)
+
+========================================================================
+GPL v2 licenses
+========================================================================
+The following components are provided under the GPL v2 license. See project links for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+
+    (GPL v2) JQuery (https://jquery.org/license)
```
[the remark this answers is not in the document]
Comment (nwangtw) [reply LICENSE]: GPL is scary. :(

From jquery website:  https://github.com/jquery/jquery/blob/master/LICENSE.txt

It seems it is MIT only now?

File: NOTICE (hunk 1 above)
Comment (nwangtw) [reply NOTICE]: kk. thx.

File: LICENSE (hunk 2 above)
[the remark this answers is not in the document]
Comment (nlu90) [reply LICENSE]: @sijie  @nwangtw  Here is the jquery related license info:

```
./tools/ui/resources/static/js/jquery.pailer.js:2: * Licensed under the Apache License, Version 2.0 (the "License");
./shell/assets/jquery.pailer.js:2: * Licensed under the Apache License, Version 2.0 (the "License");
./shell/assets/jquery.js:6: * Dual licensed under the MIT or GPL Version 2 licenses.
```

The last line says it's "Dual licensed" and those source code doesn't contain any version info.

[the remark this answers is not in the document]
Comment (sijie) [reply LICENSE]: @nlu90 for dual licensed, just pick the one that is compatible with ALv2. so MIT is good enough.

**Pull request merged**